packages/node/src/nodeClient.ts

```diff
@@ -46,35 +47,50 @@ export class NodeClient implements Client<Options> {
     // Check if there's 0 events, flush is not needed.
     const arrayLength = this._events.length;
```

Comment:
Not related to this PR changes, but is `arrayLength` necessary? I feel it's redundant with just using `this._events.length` directly